### PR TITLE
Add button on popup to directly see stats

### PR DESCRIPTION
**Pull request by KillianKemps**
Allow to directly go to the preferences from the popup button.

### CODE REVIEW

File: lib/popup/popup.vue [hunk 1]
```diff
@@ -122,6 +123,18 @@ export default {
...
+    seeStats: function() {
+      function onOpened() {
+        console.log(`Options page opened`);
+      }
+
+      function onError(error) {
+        console.log(`Error: ${error}`);
+      }
+
+      var opening = browser.runtime.openOptionsPage();
```
Comment (elegaanz) [inline lib/popup/popup.vue]: Prefer `const`.

File: lib/popup/popup.vue [hunk 2]
```diff
@@ -122,6 +123,18 @@ export default {
...
+      opening.then(onOpened, onError);
```
Comment (elegaanz) [inline lib/popup/popup.vue]: I think it's more explicit to use `catch`.

**Review by elegaanz — DISMISSED** [review]
I'm not sure you really need these two callback in the `seeStats` function, you could just call `openOptionsPage`.

File: lib/popup/popup.vue (hunk 2 above)
Comment (KillianKemps) [reply lib/popup/popup.vue]: I replaced with `catch` :)

File: lib/popup/popup.vue (hunk 1 above)
Comment (KillianKemps) [reply lib/popup/popup.vue]: Changed to `const`

**Comment by elegaanz** [comment]
Maybe you could just use an arrow function to log errors, but thé code looks good.

**Comment by elegaanz** [comment]
Btw, I just noticed that ESLint didn't failed while it should have, because you are using semi-colons… I'll need to see why. :thinking:

**Comment by elegaanz** [comment]
The test are not passing on Travis because we don't have a lockfile, so a more recent Vue version is installed, and it don't works with our current tests. But I fixed this problem in #29, and because it will be merged before this PR, you don't have to worry. But as I said in #30, we should probably use yarn to avoid these kind of problems in the future.

**Review by elegaanz — APPROVED** [review]
Looks good. Feel free to merge (once the tests are OK).

**Comment by elegaanz** [comment]
Well, Travis seems to be blocked on `npm install`. https://travis-ci.org/Qowala/browser-extension/jobs/308624512

I guess we can safely merge this…

**Comment by KillianKemps** [comment]
Well, Travis CI may not have like the several force push I did in a stretch :D

**Comment by KillianKemps** [comment]
Finally the master branch don't pass the tests: https://travis-ci.org/Qowala/browser-extension :D 
Some kind of timeout issue with an async function?

**Comment by elegaanz** [comment]
I don't know. It looks like npm is broken. We will see if will work better with yarn, as I will set it up soon (like tomorrow maybe).

**Pull request merged**